### PR TITLE
another pep exception

### PR DESCRIPTION
**Pull request by agnesk92**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,8 +16,9 @@ Each chapter takes the form of a list of rules, followed by subsections of clari
 Finally, there are sometimes good reasons for breaking the rules, so use your best judgement.
 
 ## Style
-1. PEP8 is **law**, never break PEP8. Only exception is:
+1. PEP8 is **law**, never break PEP8. Only exceptions:
     - lines are allowed to be up to 99 characters.
+    - functions with long argument list should prefer [Black style](https://github.com/psf/black/issues/1178)
```
Comment (cristianbercu) [inline README.md:21]: If I am understanding this correctly, you want to recommend that a function like this:

```python
def sum(a: int, b: int ... , x: int) -> None:
    return a + b + .. + x
```

should be styled like this:
```python
def sum(
    a: int,
    b: int,
    ... ,
    x: int
) -> None:
    return a + b + .. + x
```

Makes sense 👍🏼 . However, the guideline is for lecturing and ideally, any reader shouldn't be redirected to another webpage to make sense of it. Think of it as a scientific article where every statement backed by another scientific reading is re-iterated by the author(s), marked with a superscript number, and referenced in the end; take Wikipedia as an example.

I think it's best if you provide here an example of what is the black style for a long argument list and not reference the Github issue, but the Black documentation.

File: README.md [hunk 2]
```diff
@@ -16,8 +16,9 @@ Each chapter takes the form of a list of rules, followed by subsections of clari
 Finally, there are sometimes good reasons for breaking the rules, so use your best judgement.
 
 ## Style
-1. PEP8 is **law**, never break PEP8. Only exception is:
+1. PEP8 is **law**, never break PEP8. Only exceptions:
```
Comment (cristianbercu) [inline README.md:19]:
```suggestion
1. PEP8 is **law**, never break PEP8. The only exceptions are:
```

😄

**Review by cristianbercu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cristianbercu) [reply README.md:21]: We violate it which is why you point it out as an exception. Isn't that what you are trying to say here??

_functions with long argument list should prefer Black style_

**Review by cristianbercu — CHANGES_REQUESTED** [review]
I don't see any changes here, not sure why re-requesting a review. In the comments, there's not enough information to support an entry to the guideline

**Comment by agnesk92** [comment]
> I don't see any changes here, not sure why re-requesting a review. In the comments, there's not enough information to support an entry to the guideline

https://github.com/GameAnalytics/python-style-guide/pull/5/commits/e2f426dfe1aefd076b271ca4c588c9d812bca4ed

**Review by cristianbercu — APPROVED** [review]

**Pull request merged**